pipeline/dags/fetch_binding_ids.py

```diff
@@ -13,7 +13,10 @@
 from harvester.pmh_interface import PMH_API
 
 
-SET_IDS = ["col-361", "col-501", "col-24", "col-82", "col-861"]
+SET_IDS = [
+    # "col-361",
```

Comment:
I think this might be a source of annoyance in the future when we want to test with col-361, enable in it the download dag, forget about this, and have to debug. Could this be left in, since it doesn't cost much (or better yet, make a joint configuration for them, although I know that's more work, so maybe not here and now)?

Reply:
Good point, this collection is so small that it doesn't matter if the ID's get fetched in production. Uncommented the line.